execute_pipeline.py

```diff
@@ -82,6 +109,18 @@ def _CreateArgumentParser():
       type=str,
       default="..",
       help="Root directory where the input, output, and tool repositories live")
+  parser.add_argument(
+      "--local_repo",
+      type=str,
+      default=None,
+      help="Directory where local proto and gapic configs lives. This flag is "
+           "used for remote execution only.")
+  parser.add_argument(
+        "--env",
+        type=str,
+        default="LOCAL",
+        help="Specify which environment should be used to execute pipeline. "
+             "Valid values are 'LOCAL' and 'REMOTE'")
```

Comment:
I am doing this to support multiple environments. Eventually, we will have limited environment choices. It now can be arbitrary value for better local testing purpose. Developer can start a conductor locally (via `python start_conductor.py --jobboard_name=ethan` and test the change in another console by specify '--env=ethan').